model/perf.go

```diff
@@ -436,3 +436,34 @@ func (r *PerformanceResults) findAllChildrenGraphLookup(parent string, maxDepth
 	}
 	return nil
 }
+
+// Returns performance results with missing or outdated rollup information for
+// the given `name` and `version`.
+func (r *PerformanceResults) FindOutdatedRollups(name string, version int, after time.Time) error {
+	conf, session, err := cedar.GetSessionWithConfig(r.env)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	defer session.Close()
+
+	search := bson.M{
+		perfCreatedAtKey: bson.M{"$gt": after},
+		bsonutil.GetDottedKeyName(perfArtifactsKey, artifactInfoFormatKey): FileFTDC,
+		"$or": []bson.M{
+			{
+				bsonutil.GetDottedKeyName(perfRollupsKey, perfRollupsStatsKey): bson.M{
+					"$elemMatch": bson.M{
+						perfRollupValueNameKey:    name,
+						perfRollupValueVersionKey: bson.M{"$lt": version},
```

Comment:
nit: lte?

Reply:
We don't want to update a rollup if it has the current version, it should be strictly less than.